can-bus/json/FSM/FSM_enum.json

```diff
@@ -1,3 +1,73 @@
 {
-    
+	"BrakeActuation":{
+		"BRAKE_IS_ACTUATED_FALSE_CHOICE": 0,
+		"BRAKE_IS_ACTUATED_TRUE_CHOICE": 1
+	},
+	"BrakePressureSensorOCSC":{
+		"PRESSURE_SENSOR_IS_OPEN_OR_SHORT_CIRCUIT_FALSE_CHOICE": 0,
+		"PRESSURE_SENSOR_IS_OPEN_OR_SHORT_CIRCUIT_TRUE_CHOICE": 1
+	},
+	"BrakePedalOCSC":{
+		"PEDAL_IS_OPEN_OR_SHORT_CIRCUIT_FALSE_CHOICE": 0,
+		"PEDAL_IS_OPEN_OR_SHORT_CIRCUIT_TRUE_CHOICE": 1
+	},
+	"PappsOCSC":{
+		"PAPPS_OCSC_IS_ACTIVE_FALSE_CHOICE": 0,
+		"PAPPS_OCSC_IS_ACTIVE_TRUE_CHOICE": 1
+	},
+	"SappsOCSC":{
+		"SAPPS_OCSC_IS_ACTIVE_FALSE_CHOICE": 0,
+		"SAPPS_OCSC_IS_ACTIVE_TRUE_CHOICE": 1
+	},
+	"AppsDisagreement": {
+		"APPS_HAS_DISAGREEMENT_FALSE_CHOICE": 0,
+		"APPS_HAS_DISAGREEMENT_TRUE_CHOICE": 1
+	},
+	"BrakeAccDisagreement":{
+		"BRAKE_ACC_DISAGREEMENT_FALSE_CHOICE": 0,
+		"BRAKE_ACC_DISAGREEMENT_TRUE_CHOICE": 1
+	},
+	"FlowMeterHasUnderFlow":{
+		"FLOW_METER_HAS_UNDERFLOW_FALSE_CHOICE": 0,
+		"FLOW_METER_HAS_UNDERFLOW_TRUE_CHOICE": 1
+	},
+	"SteeringAngleSensorOCSC":{
+		"STEERING_SENSOR_OCSC_FALSE_CHOICE": 0,
+		"STEERING_SENSOR_OCSC_TRUE_CHOICE": 1
+	},
+	"LeftWheelSpeedOutOfRange": {
+		"LEFT_WHEEL_SPEED_OUT_OF_RANGE_OK_CHOICE": 0,
+		"LEFT_WHEEL_SPEED_OUT_OF_RANGE_UNDERSPEED_CHOICE": 1,
+		"LEFT_WHEEL_SPEED_OUT_OF_RANGE_OVERSPEED_CHOICE": 2
+	},
+	"RightWheelSpeedOutOfRange": {
+		"RIGHT_WHEEL_SPEED_OUT_OF_RANGE_OK_CHOICE": 0,
+		"RIGHT_WHEEL_SPEED_OUT_OF_RANGE_UNDERSPEED_CHOICE": 1,
+		"RIGHT_WHEEL_SPEED_OUT_OF_RANGE_OVERSPEED_CHOICE": 2
+	},
+	"SteeringAngleOutOfRange": {
+		"STEERING_ANGLE_OUT_OF_RANGE_OK_CHOICE": 0,
+		"STEERING_ANGLE_OUT_OF_RANGE_WIDELEFT_CHOICE": 1,
+		"STEERING_ANGLE_OUT_OF_RANGE_WIDERIGHT_CHOICE": 2
+	},
+	"FrontBrakePressureOutOfRange": {
+		"BRAKE_FRONT_PRESSURE_OUT_OF_RANGE_OK_CHOICE": 0,
+		"BRAKE_FRONT_PRESSURE_OUT_OF_RANGE_UNDER_CHOICE": 1,
+		"BRAKE_FRONT_PRESSURE_OUT_OF_RANGE_OVER_CHOICE": 2
+	},
+	"RearBrakePressureOutOfRange": {
+		"BRAKE_REAR_PRESSURE_OUT_OF_RANGE_OK_CHOICE": 0,
+		"BRAKE_REAR_PRESSURE_OUT_OF_RANGE_UNDER_CHOICE": 1,
+		"BRAKE_REAR_PRESSURE_OUT_OF_RANGE_OVER_CHOICE": 2
+	},
+	"FlowRateOutOfRange": {
+		"FLOW_RATE_OUT_OF_RANGE_OK_CHOICE": 0,
+		"FLOW_RATE_OUT_OF_RANGE_UNDERFLOW_CHOICE": 1,
+		"FLOW_RATE_OUT_OF_RANGE_OVERFLOW_CHOICE": 2
+	},
```

Comment:
No need to make a separate enum for each of these signals, can you declare a single RangeCheck enum in the shared code? Which takes RANGE_CHECK_OK, RANGE_CHECK_UNDERFLOW, RANGE_CHECK_OVERFLOW.
I think other boards use range checking so useful to make it shared.